package.json

```diff
@@ -84,7 +84,8 @@
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.20.0 <13.0.0"
+    "node": ">=10.20.0 <13.0.0",
+    "npm": "<7.0.0"
```

Comment:
This results in a warning being printed if users attempt to install the CLI with npm v7:
```
$ node --version
v12.21.0

$ npm --version
7.6.3

$ npm install
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'balena-cli@12.41.2',
npm WARN EBADENGINE   required: { node: '>=10.20.0 <13.0.0', npm: '<7.0.0' },
npm WARN EBADENGINE   current: { node: 'v12.21.0', npm: '7.6.3' }
npm WARN EBADENGINE }
...
```